src/devices/Nmea0183/Sentences/EngineData.cs

```diff
@@ -0,0 +1,98 @@
+﻿// Licensed to the .NET Foundation under one or more agreements.
+// The .NET Foundation licenses this file to you under the MIT license.
+
+using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Text;
+using System.Threading.Tasks;
+using UnitsNet;
+
+namespace Iot.Device.Nmea0183.Sentences
+{
+    /// <summary>
+    /// A helper DTO to transfer engine data in one blob
+    /// </summary>
+    [Serializable]
```

Comment:
why do we need the attribute?

Reply:
Because DTOs should generally have this attribute.